auth_saml/__manifest__.py

```diff
@@ -4,13 +4,13 @@
 
 {
     "name": "SAML2 Authentication",
-    "version": "16.0.1.0.2",
+    "version": "17.0.1.0.2",
```

Comment:
Version should be reset to *17.0.1.0.0*.

Reply:
Done.